.github/styles/kong/We.yml

```diff
@@ -4,8 +4,6 @@ link: 'https://developers.google.com/style/pronouns#personal-pronouns'
 level: warning
 ignorecase: true
 tokens:
-  - we
-  - we'(?:ve|re)
   - ours?
   - us
```

Comment:
Honestly, I think we should get rid of this rule entirely, not just "we". There's still a lot of exceptions where "our/ours", "us" are valid.

Can we also get rid of the "I" rule? https://github.com/Kong/docs.konghq.com/blob/vale-standardize-on-plugins/.github/styles/kong/First-person.yml